README.md

```diff
@@ -24,14 +23,69 @@ If you have downloaded the repo without using the `--recurse-submodules` argumen
 git submodule update --init --recursive
 ```
 
+## MQTT Config File
+
+The MQTT Agent library exposes build configuration macros that are required for building the library.
+However, the MQTT Agent library does not include its own configuration file, instead inheriting any macros defined in coreMQTT's `core_mqtt_config.h`. Lists of configurable macros and their default values can be found at the top of [core_mqtt_agent.h](source/include/core_mqtt_agent.h) and [core_mqtt_agent_command_functions.h](source/include/core_mqtt_agent_command_functions.h).
```

Comment:
Suggestion for better expression of the config macros specific to the Agent Library, and other configs indirectly inherited from coreMQTT/
```suggestion
However, the MQTT Agent library does not include its own configuration file, but exposes configurable macros and their default values at the top of [core_mqtt_agent.h](source/include/core_mqtt_agent.h) and [core_mqtt_agent_command_functions.h](source/include/core_mqtt_agent_command_functions.h). Also, the library inherits any macros defined in coreMQTT's `core_mqtt_config.h`. ```

Reply:
I'm assuming the "```" is a typo so I will edit your comment